docs/en/advanced_guides/dataset.md

```diff
@@ -2,85 +2,32 @@
 
 ## Supported Data Format
 
-### Image Super-Resolution
-
-- [SRAnnotationDataset](/mmedit/datasets/sr_annotation_dataset.py)
-  General paired image dataset with an annotation file for image restoration.
-- [SRFolderDataset](/mmedit/datasets/sr_folder_dataset.py)
-  General paired image folder dataset for image restoration.
-- [SRFolderGTDataset](/mmedit/datasets/sr_folder_gt_dataset.py)
-  General ground-truth image folder dataset for image restoration, where low-quality image should be generated in pipeline.
-- [SRFolderRefDataset](/mmedit/datasets/sr_folder_ref_dataset.py)
-  General paired image folder dataset for reference-based image restoration.
-- [SRLmdbDataset](/mmedit/datasets/sr_lmdb_dataset.py)
-  General paired image lmdb dataset for image restoration.
-- [SRFacialLandmarkDataset](/mmedit/datasets/sr_facial_landmark_dataset.py)
-  Facial image and landmark dataset with an annotation file.
-
-### Video Super-Resolution
-
-- [SRFolderMultipleGTDataset](/mmedit/datasets/sr_folder_multiple_gt_dataset.py)
-  General dataset for video super resolution, used for recurrent networks.
-- [SRREDSDataset](/mmedit/datasets/sr_reds_dataset.py)
-  REDS dataset for video super resolution.
-- [SRREDSMultipleGTDataset](/mmedit/datasets/sr_reds_multiple_gt_dataset.py)
-  REDS dataset for video super resolution for recurrent networks.
-- [SRTestMultipleGTDataset](/mmedit/datasets/sr_test_multiple_gt_dataset.py)
-  Test dataset for video super resolution for recurrent networks.
-- [SRVid4Dataset](/mmedit/datasets/sr_vid4_dataset.py)
-  Vid4 dataset for video super resolution.
-- [SRVimeo90KDataset](/mmedit/datasets/sr_vimeo90k_dataset.py)
-  Vimeo90K dataset for video super resolution.
-- [SRVimeo90KMultipleGTDataset](/mmedit/datasets/sr_vimeo90k_multiple_gt_dataset.py)
-  Vimeo90K dataset for video super resolution for recurrent networks.
-
-### Video Frame Interpolation
-
-- [VFIVimeo90KDataset](/mmedit/datasets/vfi_vimeo90k_dataset.py)
-  Vimeo90K dataset for video frame interpolation.
-
-### Matting
-
+- [BasicImageDataset](/mmedit/datasets/basic_image_dataset.py)
```

Comment:
List basic datasets without introduction seems abrupt. Maybe you can say: We support three types of data, image, frames, xxx, and customized dataset are supposed to inherit from one of them xxx.